gitwebhookproxy.go

```diff
@@ -14,7 +14,8 @@ var (
 	flagSet       = flag.NewFlagSetWithEnvPrefix(os.Args[0], "GWP", 0)
 	listenAddress = flagSet.String("listen", ":8080", "Address on which the proxy listens.")
 	upstreamURL   = flagSet.String("upstreamURL", "", "URL to which the proxy requests will be forwarded (required)")
-	secret        = flagSet.String("secret", "", "Secret of the Webhook API (required)")
+	secret        = flagSet.String("secret", "", "Secret of the Webhook API")
+	validate      = flagSet.Bool("validate", true, "Are requests validated using secret. Needs to be set to false if no secret is set in provider.")
```

Comment:
@msvticket I think we could just make the secret optional instead of adding a new flag for validation ? @waseem-h

Reply:
OK, I have done like that now.